crates/wildbg-c/src/lib.rs

```diff
@@ -8,9 +8,7 @@ use logic::cube::CubeInfo;
 use logic::wildbg_api::{WildbgApi, WildbgConfig};
 
 // When this file is changed, recreate the header file by executing this from the project's root:
-// touch cbindgen.toml
-// cbindgen --config cbindgen.toml --crate wildbg-c --output crates/wildbg-c/wildbg.h --lang c
-// rm cbindgen.toml
+// cbindgen --config crates/wildbg-c/bindgen.toml  --crate wildbg-c --output crates/wildbg-c/wildbg.h --lang c
```

Comment:
There is a typo: `cbindgen.toml` is missing the `c`. The action fails because of that.

```suggestion
// cbindgen --config crates/wildbg-c/cbindgen.toml  --crate wildbg-c --output crates/wildbg-c/wildbg.h --lang c
```